### DISCUSSION TITLE
Is it possible to combine click and argparse?

### DISCUSSION THREAD

**Discussion by GlenNicholls** [pallets/click#2413] [Q&A]
I have an app that wraps a pypi package and extends its argparse CLI for one of the modes of operation. However, I would like to use click instead and I'm wondering if there's a way to add those args/options to our click interface from that argparse object? Any examples or documentation that cover this or a similar subject would be really helpful.

I found [click.BaseCommand](https://github.com/pallets/click/discussions/2413), but it's not clear to me how to combine argparse and a click implementation.

**Comment by davidism** [comment]
Yes, you can override `Command.parse_args` to do whatever processing you want.
